CryptoPkg/Library/OpensslLib/OpensslGen/include/openssl/bio.h

```diff
@@ -867,7 +867,7 @@ int BIO_meth_set_puts(BIO_METHOD *biom,
                       int (*puts) (BIO *, const char *));
 int (*BIO_meth_get_gets(const BIO_METHOD *biom)) (BIO *, char *, int);
 int BIO_meth_set_gets(BIO_METHOD *biom,
-                      int (*gets) (BIO *, char *, int));
+                      int (*ossl_gets) (BIO *, char *, int));

```

Comment:
Is there any reason this function got renamed ? Does this come from a change in openssl upstream if so could you let us know which change exactly ?

Reply:
Please refer to this: https://github.com/openssl/openssl/issues/22296